template/src/App.tsx

```diff
@@ -1,17 +1,18 @@
 import React from 'react'
 
-import logo from './logo.svg'
-import './App.css'
+import logo from './assets/images/logo.svg'
+import './dummy.scss'
+import './assets/stylesheets/application.scss'
```

Comment:
Ditto

Reply:
@noor-ahmed I have already created an issue for this purpose - https://github.com/nimblehq/react-templates/issues/22
I was planning to configure it in a different PR. Is that okay?

Reply:
Thats also nice. ✌🏻